client/telecomUniverseComponents/telecom/voip/service/alias/voip-service-alias.service.js

```diff
@@ -161,16 +186,152 @@ export default class {
       billingAccount,
       featureType: 'redirect',
       serviceName,
-    }, { destination }).$promise.then(task => this.voipServiceTask
+    }, { destination }).$promise.then(({ taskId }) => this.voipServiceTask
       .startPolling(
         billingAccount,
         serviceName,
-        task.taskId,
+        taskId,
         {
           namespace: `redirectChangeDestinationTask_${serviceName}`,
           interval: 1000,
           retryMaxAttempts: 0,
         },
       )).catch(error => error);
   }
+
+  /**
+   *  @ngdoc method
+   *  @name managerApp.service:voipServiceAlias#fetchConferenceNumber
+   *  @methodOf managerApp.service:voipServiceAlias
+   *
+   *  @description
+   *  <p>Fetch conference number.</p>
+   *
+   *  @param  {VoipService} number (destructured) The given VoipService number.
+   *
+   *  @return {VoipService}                       The conference number
+   */
+  fetchConferenceNumber({ billingAccount, serviceName }) {
+    return this.OvhApiTelephony.Conference().v6().settings({
+      billingAccount,
+      serviceName,
+    }).$promise;
+  }
+
+  /**
+   *  @ngdoc method
+   *  @name managerApp.service:voipServiceAlias#updateConferenceNumberSettings
+   *  @methodOf managerApp.service:voipServiceAlias
+   *
+   *  @description
+   *  <p>Update settings of the conference number.</p>
+   *
+   *  @param  {VoipService} number (destructured) The given VoipService number.
+   *  @param  {Object}      settings              New conference settings
+   *
+   */
+  updateConferenceNumberSettings({ billingAccount, serviceName }, settings) {
+    return this.OvhApiTelephony.Conference().v6().updateSettings({
+      billingAccount,
+      serviceName,
+    }, settings).$promise;
+  }
+
+  /**
+   *  @ngdoc method
+   *  @name managerApp.service:voipServiceAlias#uploadConferenceNumberAnnounce
+   *  @methodOf managerApp.service:voipServiceAlias
+   *
+   *  @description
+   *  <p>Upload an announce file for the conference number.</p>
+   *
+   *  @param  {VoipService} number (destructured) The given VoipService number.
+   *  @param  {Object}      file                  The new file to upload
+   *
+   */
+  uploadConferenceNumberAnnounce({ billingAccount, serviceName }, file) {
+    return this.OvhApiMe.Document().v6().upload(file.name, file)
+      .then(({ id }) => this.OvhApiTelephony.Conference().v6().announceUpload({
+        billingAccount,
+        serviceName,
+      }, { documentId: id }).$promise)
+      .then(({ taskId }) => this.voipServiceTask
+        .startPolling(
+          billingAccount,
+          serviceName,
+          taskId,
+          {
+            namespace: `uploadConferenceNumberAnnounceTask_${serviceName}`,
+            interval: 1000,
+            retryMaxAttempts: 0,
+          },
+        ))
+      .then(() => true)
+      .catch(error => error);
```

Comment:
Is this intentional ? Handling errors in `then` is not a good idea, except maybe if the errors returned by the API are actually not errors